website/docs/quick-start/installation/modal/app.py

```diff
@@ -32,17 +32,21 @@ def download_model():
     .pip_install("asgi-proxy-lib")
 )
 
-stub = Stub("tabby-server-" + MODEL_ID.split("/")[-1], image=image)
+app = App("tabby-server-" + MODEL_ID.split("/")[-1], image=image)
 
+ee_volume = Volume.from_name("tabby-ee-vol", create_if_missing=True)
+ee_dir = "/data/ee"
```

Comment:
consider adding the entier `/data` directory to volume, and use https://github.com/TabbyML/tabby/blob/7b048c2e1a74271d91c2c758ee84c877e2bc11e2/website/blog/2024-03-25-deploy-tabby-in-air-gapped-environment-with-docker/index.md?plain=1#L35 to separate dir for model cache?

Reply:
adding the entire /data directory to volume.